src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -1674,8 +1669,8 @@ sourceFile, getSourceStorageAccountPrimarySharedKey()))
   }
 
   @Test
-  @Ignore("Ignoring due to flakiness and deprioritization of Azure")
-  public void testDatasetFileRefValidation() throws Exception {
+  @Disabled("Ignoring due to flakiness and deprioritization of Azure")
```

Comment:
Ok, so the @Disabled flag should now work now that it has been converted to Junit5?

Reply:
Yes this works as expected. I didn't realize we tried it before now.